vim/merlin/autoload/merlin.vim

```diff
@@ -270,6 +270,22 @@ function! merlin#TypeOfSel()
   call merlin#TypeOf(s:get_visual_selection())
 endfunction
 
+function! s:type_at_balloon()
+	MerlinPy vim.command("let l:type = " + merlin.vim_type_enclosing_at_pos(
+				\ vim.vvars['beval_bufnr'], vim.vvars['beval_lnum'], vim.vvars['beval_col']))
```

Comment:
I think I'd rather see the following function in `merlin.py`:
```python
def vim_type_enclosing_at_mouse():
  bufnr = vim.vvars['beval_bufnr']
  if bufnr != vim.current.buffer.number:
        return '{}'
  line = vim.vvars['beval_lnum']
  col = vim.vvars['beval_col']
  return type_enclosing_at_pos(line, col)
```

And then this line can be changed to
```
MerlinPy vim.command("let l:type = " + merlin.vim_type_enclosing_at_mouse())
```

Would there be an issue with that?

Reply:
That seems better, i'll try it.